Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM gcr.io/tekton-releases/dogfooding/tkn@sha256:f69a02ef099d8915e9e4ea1b74e43b7a9309fc97cf23cb457ebf191e73491677 as tkn
```

Comment:
shall we use the tkn images @rupalibehera is building.

this can be changed in the future as well. As we will be asking soulution architects and other contact persons to use our must gather on customer portals, our own images will be a better choice.

Reply:
oh i see this is a multistage build.. 
however, let us try to copy it from our own images if available 🧑‍💻

Reply:
👍🏻 always use our own images, as this is hard req for most of the things we build!

Reply:
I was unaware of that but yes we can use our own images :+1:. Will ask @rupalibehera for that